### PR TITLE
Added page titles and auto scrolling to top of page on load

### PR DESCRIPTION
**Pull request by jupiter007**
Fixes [#559](https://github.com/CDLUC3/dmptool/issues/559) .

Changes proposed in this PR:
- Added page wrapper that allows us to add a page title to the Upload pages and to auto scroll user to top of page when the page loads.
- My "Pretty" formatter added a new line to several of the tinymce skin files.

### CODE REVIEW

File: react-client/src/pages/dashboard/dashboard.js
Comment (briri) [inline react-client/src/pages/dashboard/dashboard.js:1]: It looks like you just added the PageWrapper here and then maybe the Pretty formatter changed up the spacing. Is that true?

File: react-client/src/pages/plan/overview/overview.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { useContext, useEffect, useState } from "react";
```
Comment (briri) [inline react-client/src/pages/plan/overview/overview.js:1]: thanks for cleaning up these unused imports

**Review by briri — APPROVED** [review]
This looks great. Thanks for adding this @jupiter007. I think it will make user's experience's better when navigating in the tool

File: react-client/src/pages/dashboard/dashboard.js
Comment (jupiter007) [reply react-client/src/pages/dashboard/dashboard.js:1]: Yes, the only update I made was adding the PageWrapper, but some added spacing made it difficult to see on the "diff".

**Pull request merged**